components/ca-crud.html

```diff
@@ -729,6 +743,20 @@
                         self.onaction(action, data, dlg);
                         return false;
                     }
+
+                    // find the navigation for the button that was clicked
+                    const nav = labelToNav[button];
+
+                    if (nav) {
+                        // we want to raise a navigation action event
+                        const event = new CustomEvent('ca-crud-action-nav', {
+                            detail: data,
+                            bubbles: true
+                        });
+                        self.dispatchEvent(event);
+                        return true;
```

Comment:
No it's a good comment. But as with all other if blocks in this area, we are explicitly returning a boolean false for action, previewable, deletable, editable but that is because we explicitly want to keep the dialog open. When navigating away, we explicitly want to close the dialog and I didn't want to rely on the return true at the bottom of the callback.